scripts/jest/preprocessor.js

```diff
@@ -79,9 +82,12 @@ module.exports = {
       if (process.env.NODE_ENV === 'development' && !isInDevToolsPackages) {
         sourceOnlyPlugins.push(pathToBabelPluginReplaceConsoleCalls);
       }
-      const plugins = (isTestFile ? testOnlyPlugins : sourceOnlyPlugins).concat(
+      let plugins = (isTestFile ? testOnlyPlugins : sourceOnlyPlugins).concat(
         babelOptions.plugins
       );
+      if (isTestFile && isInDevToolsPackages) {
+        plugins = plugins.concat(devtoolsPlugins);
```

Comment:
nit: If you wanted to, I think you could leave `plugins` a const and just `push` to it– since the `concat` call above already created a new `plugins` array.